__tests__/backend/mainController.tests.mjs

```diff
@@ -0,0 +1,19 @@
+import mainController from '../../server/controllers/mainController.mjs';
+import supertest from 'supertest';
+import express from 'express';
+
+const app = express();
+
+app.get('/', mainController.get);
+
+describe('Main Controller', () => {
+    test('should return the correct JSON response', async () => {
+        const response = await supertest(app)
+            .get('/')
+            .expect(200);
+
+        expect(response.body).toEqual({
+            data: 'This is a full stack app!'
+        });  // Verify that the response contains the correct data
+    });
+});
```

Comment:
Do we need to add test file for mainController.mjs file?

Reply:
I will remove this test as it is not relevant to our application.